services/tier-service/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/dal/ddb/TierAttribute.java

```diff
@@ -15,7 +31,10 @@ public enum TierAttribute {
             (tierBuilder, attributeValue) -> tierBuilder.name(attributeValue.s())),
     description(tier -> AttributeValue.builder().s(tier.getDescription()).build(),
             attributeValue -> !Utils.isEmpty(attributeValue.s()),
-            (tierBuilder, attributeValue) -> tierBuilder.description(attributeValue.s()));
+            (tierBuilder, attributeValue) -> tierBuilder.description(attributeValue.s())),
+    defaultTier(tier -> AttributeValue.builder().bool(tier.defaultTier()).build(),
```

Comment:
Sort of a style thing, but comes from usability concerns. In all the other services, we don't use camelCase field names in the database. Instead we convert to snake_case to avoid confusion and case sensitivity issues. We don't have to change this, but for consistency we should consider it. In the future we're also going to end up with a billingPlanId -- billing_plan_id in the tiers table too.